src/Spout/Writer/Internal/XLSX/Workbook.php

```diff
@@ -62,13 +62,18 @@ public function __construct($tempFolder, $shouldUseInlineStrings, $shouldCreateN
     /**
      * Creates a new sheet in the workbook. The current sheet remains unchanged.
      *
+     * @param string $sheetName The custom name of the sheet
      * @return Worksheet The created sheet
      * @throws \Box\Spout\Common\Exception\IOException If unable to open the sheet for writing
      */
-    public function addNewSheet()
+    public function addNewSheet($sheetName = null)
```

Comment:
I would rather keep this function as is and add a function of the sheet itself to update the name:

```php
$firstSheet = $writer->getCurrentSheet()->setName('Custom name');

$newSheet = $writer->addNewSheetAndMakeItCurrent();
$newSheet->setName('My Sheet');
```